### PR TITLE
Total English Retranslation

### PR DESCRIPTION
**Pull request by glyphical**
If I'm doing this correctly, this branch contains a total English retranslation of the mod to give it perfect English in a consistent, professional style. In many cases, text was shortened and made easier to understand for end users. I realized I did the brackets wrong in the translation file I provided previously, so that has been fixed.

### CODE REVIEW

**Comment by CoolLKKPS** [comment]
hello, i will take a look of this PR, hang on

File: AntiCheat/Lang/en_US.resx [hunk 1]
```diff
@@ -238,118 +238,118 @@
...
-    <value>Detected player {player} instantly killed a monster! ({enemyName}, monster HP before: {HP})</value>
+    <value>{player} was detected killing a monster instantly. ({enemyName}, monster HP before: {HP})</value>
   </data>
   <data name="msg_Landmine" xml:space="preserve">
-    <value>Detected player {player} anomaly triggered the landmine!</value>
+    <value>{player} was detected forcing a landmine explosion.</value>
   </data>
   <data name="msg_Map" xml:space="preserve">
-    <value>Detected player {player} using Mini-Map mod!</value>
+    <value>{player} was detected to have a mini-map mod. This can make finding loot much easier.</value>
```
Comment (CoolLKKPS) [inline AntiCheat/Lang/en_US.resx:295]: yea this part is modified by me, but still u need to remove additional message to only keep original meaning

File: AntiCheat/Lang/en_US.resx [hunk 2]
```diff
@@ -238,118 +238,118 @@
...
-    <value>Detected player {player} using {item} without respect the cooldown!</value>
+    <value>{player} was detected using an item faster than its cooldown would allow.</value>
   </data>
   <data name="msg_Jetpack" xml:space="preserve">
-    <value>Detected player {player} anomaly exploded the jetpack!</value>
+    <value>{player} was detected forcing a jetpack explosion.</value>
   </data>
   <data name="msg_Kick" xml:space="preserve">
-    <value>{player} was kicked!</value>
+    <value>{player} was banished!</value>
```
Comment (CoolLKKPS) [inline AntiCheat/Lang/en_US.resx:286]: No, keep original meaning

File: AntiCheat/Lang/en_US.resx [hunk 3]
```diff
@@ -238,118 +238,118 @@
...
-    <value>Detected player {player} destroying items!</value>
+    <value>{player} was detected destroying items.</value>
   </data>
   <data name="msg_Enemy_ChangeOwnershipOfEnemy" xml:space="preserve">
-    <value>Detected player {player} attempting to change the monster behavior!</value>
+    <value>{player} was detected trying to control a monster.</value>
   </data>
   <data name="msg_Enemy_SwitchToBehaviour" xml:space="preserve">
-    <value>Detected player {player} attempting to change the monster mechanism!</value>
+    <value>{player} was detected trying to alter a monster's behavior.</value>
```
Comment (CoolLKKPS) [inline AntiCheat/Lang/en_US.resx:250]: im not sure this meaning, so for example the jester has 3 stages, if cheater try to change that will receive this message
im not sure how to say that correctly so i use mechanism word instead

File: AntiCheat/Lang/en_US.resx [hunk 4]
```diff
@@ -238,118 +238,118 @@
...
-    <value>Detected player {player} tried to spoof {player2}'s name!</value>
+    <value>{player} tried to spoof {player2}'s name in chat.</value>
   </data>
   <data name="msg_DespawnItem" xml:space="preserve">
-    <value>Detected player {player} destroying items!</value>
+    <value>{player} was detected destroying items.</value>
   </data>
   <data name="msg_Enemy_ChangeOwnershipOfEnemy" xml:space="preserve">
-    <value>Detected player {player} attempting to change the monster behavior!</value>
+    <value>{player} was detected trying to control a monster.</value>
```
Comment (CoolLKKPS) [inline AntiCheat/Lang/en_US.resx:247]: its not fully mean the player try to control the monster actually, cheater can make every monster to not targeting any player, that's not Control Company can do, its more than that

File: AntiCheat/Lang/en_US.resx [hunk 5]
```diff
@@ -238,118 +238,118 @@
     <value>Detected player {player} attempting to call the boss attacking code functions!</value>
   </data>
   <data name="msg_ChatReal" xml:space="preserve">
-    <value>Detected player {player} tried to spoof {player2}'s name!</value>
+    <value>{player} tried to spoof {player2}'s name in chat.</value>
```
Comment (CoolLKKPS) [inline AntiCheat/Lang/en_US.resx:241]: well i just removed "in chat" word, but its ok if really need

File: AntiCheat/Lang/en_US.resx [hunk 6]
```diff
@@ -121,100 +121,100 @@
...
-    <value>Allow players to deal normal damage when empty-handed (network delays may cause players to be empty-handed)</value>
+    <value>Allow players to deal normal damage when empty-handed. Note: Network delays may cause players to be empty-handed.</value>
   </data>
   <data name="config_SpawnWebTrap" xml:space="preserve">
-    <value>Abnormal spiderweb spawning detection</value>
+    <value>Detects when a player tries to spawn spiderwebs.</value>
   </data>
   <data name="config_Turret" xml:space="preserve">
-    <value>Abnormal turret berserk mode detection</value>
+    <value>Detects when a player forces a turret to spin.</value>
```
Comment (CoolLKKPS) [inline AntiCheat/Lang/en_US.resx:217]: again no turret spin

File: AntiCheat/Lang/en_US.resx [hunk 7]
```diff
@@ -121,100 +121,100 @@
...
-    <value>Spamming terminal noise detection</value>
+    <value>Detects the spam of terminal noise.</value>
   </data>
   <data name="config_Shovel" xml:space="preserve">
-    <value>Abnormal shovel damage detection</value>
+    <value>Detects unusual shovel damage.</value>
   </data>
   <data name="config_Shovel2" xml:space="preserve">
-    <value>Allow players to deal normal damage when empty-handed (network delays may cause players to be empty-handed)</value>
+    <value>Allow players to deal normal damage when empty-handed. Note: Network delays may cause players to be empty-handed.</value>
```
Comment (CoolLKKPS) [inline AntiCheat/Lang/en_US.resx:211]: u should make this part more easier to read, and i didn't touch this part

File: AntiCheat/Lang/en_US.resx [hunk 8]
```diff
@@ -121,100 +121,100 @@
...
-    <value>You are the host, your vote can call the autopilot.</value>
+    <value>Allow your single vote to instantly force the autopilot as host.</value>
   </data>
   <data name="config_ShipLight" xml:space="preserve">
-    <value>Spamming light switch detection</value>
+    <value>Detects when a player spawns a light switch.</value>
   </data>
   <data name="config_ShipSetting" xml:space="preserve">
-    <value>Only the host can land the ship</value>
+    <value>Makes it so that only the host may land the ship.</value>
```
Comment (CoolLKKPS) [inline AntiCheat/Lang/en_US.resx:202]: VERY ONLY the host can land the ship

File: AntiCheat/Lang/en_US.resx [hunk 9]
```diff
@@ -121,100 +121,100 @@
...
-    <value>Detect malicious lever pull behavior (kick player when this behavior is detected)</value>
+    <value>Detects when a player attempts to force the ship lever. Kicks the player if detected.</value>
   </data>
   <data name="config_ShipConfig6" xml:space="preserve">
-    <value>You are the host, your vote can call the autopilot.</value>
+    <value>Allow your single vote to instantly force the autopilot as host.</value>
   </data>
   <data name="config_ShipLight" xml:space="preserve">
-    <value>Spamming light switch detection</value>
+    <value>Detects when a player spawns a light switch.</value>
```
Comment (CoolLKKPS) [inline AntiCheat/Lang/en_US.resx:199]: this part is wrong

File: AntiCheat/Lang/en_US.resx [hunk 10]
```diff
@@ -121,100 +121,100 @@
...
-    <value>Mini-Map mod detection</value>
+    <value>Detects players using a mini-map mod, displaying the radar on the screen at all times.</value>
   </data>
   <data name="config_Mask" xml:space="preserve">
-    <value>Abnormal masked enemy detection</value>
+    <value>Detects when masked enemies are spawned abnormally.</value>
   </data>
   <data name="config_Nameless" xml:space="preserve">
-    <value>Player name detection (Nameless or Unknown)</value>
+    <value>Detects invalid or anomalous player names such as Nameless.</value>
```
Comment (CoolLKKPS) [inline AntiCheat/Lang/en_US.resx:172]: such as Nameless or Unknown

File: AntiCheat/Lang/en_US.resx [hunk 11]
```diff
@@ -121,100 +121,100 @@
...
   <data name="config_Landmine" xml:space="preserve">
-    <value>Abnormal landmine triggering detection</value>
+    <value>Detects when a player abnormally detonates a landmine.</value>
   </data>
   <data name="config_Log" xml:space="preserve">
     <value>Show log</value>
   </data>
   <data name="config_Map" xml:space="preserve">
-    <value>Mini-Map mod detection</value>
+    <value>Detects players using a mini-map mod, displaying the radar on the screen at all times.</value>
```
Comment (CoolLKKPS) [inline AntiCheat/Lang/en_US.resx:166]: remove additional part to keep original meaning

File: AntiCheat/Lang/en_US.resx [hunk 12]
```diff
@@ -121,100 +121,100 @@
...
-    <value>Abnormal gift box interaction detection</value>
+    <value>Detects players having abnormal gift box interactions.</value>
   </data>
   <data name="config_GrabObject" xml:space="preserve">
-    <value>Item pickup distance detection</value>
+    <value>Detects picking up items outside of default grab range.</value>
   </data>
   <data name="config_InfiniteAmmo" xml:space="preserve">
-    <value>Infinite ammo detection</value>
+    <value>Detects when a player has infinite ammo.</value>
```
Comment (CoolLKKPS) [inline AntiCheat/Lang/en_US.resx:142]: u need to know, if the shotgun already loaded infinite ammo, normal player can also trigger that detection

**Review by CoolLKKPS — CHANGES_REQUESTED** [review]
those issues and question u have to deal with

File: AntiCheat/Lang/en_US.resx (hunk 11 above)
Comment (glyphical) [reply AntiCheat/Lang/en_US.resx:166]: I feel like the additional information here is legitimately important. Not every player knows what a mini-map is or what it implies, and the UI has plenty of space for this info. I would like to recommend you keep this.

File: AntiCheat/Lang/en_US.resx (hunk 12 above)
Comment (glyphical) [reply AntiCheat/Lang/en_US.resx:142]: I will address this and the rest of the problems you listed momentarily. Thank you.

File: AntiCheat/Lang/en_US.resx (hunk 8 above)
Comment (glyphical) [reply AntiCheat/Lang/en_US.resx:202]: I'm not sure I understand what the correction is here. The wording is what you said--it says that with this setting enabled, only the host can land the ship.

Comment (CoolLKKPS) [reply AntiCheat/Lang/en_US.resx:202]: hmm changed to this is fine actually, ok

**Comment by glyphical** [comment]
I addressed all the comments that I could. I do recommend re-integrating the minimap mod information, but otherwise it should be prepared for use now.

**Comment by glyphical** [comment]
OK now it's ready, missed the lightswitch  issue.

**Comment by CoolLKKPS** [comment]
we don't want to criticize another mod unless it does make the game environment more terrible, we more like to send feedback to mini-map mod author to fix those problem instead add opinions in the mod; And the host should have more experience, not the mod to teach them

**Comment by glyphical** [comment]
I wouldn't say there was any criticism of the mod in my message--it simply said what minimaps do in completely neutral terms. Still I suppose I understand.

**Comment by CoolLKKPS** [comment]
but still u can add opinions in config, just don't add it in message

**Comment by glyphical** [comment]
Oh I understand now. I didn't want to keep the minimap info in the detection message, I only wanted to keep the info on what minimaps do in the config as I definitely understand removing the 'makes loot easier to find' thing. I already removed it though, it's no big deal--overall, you should be good to go with this now.

**Comment by CoolLKKPS** [comment]
will notify mod author, and i may have to modify some readme-en, thx for ur translation!

**Comment by chuxiaaaa** [comment]
Is everything ready?

**Comment by glyphical** [comment]
It's prepared for merging as far as I can tell.

**Comment by glyphical** [comment]
Just reviewed the text again and it's looking good. No typos that I saw.

**Pull request merged**